rust-ceramic/src/usage.md

```diff
@@ -0,0 +1,25 @@
+# Usage
```

Comment:
Should we call out `ceramic-one --help` here? We have good usage docs built into the binary, I'd like to refer to them here.